scripts/get-user-location.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+if [[ -z $AWS_ACCESS_KEY_ID ]] || [[ -z $AWS_SECRET_ACCESS_KEY ]] || [[ -z $AWS_SESSION_TOKEN ]]; then
+    echo "AWS session variables not set. Must set CLI short term credentials before proceeding."
+    exit 1
+fi
+
+aws_command="aws dynamodb scan"
+output_file="get-user-location-output.csv" # This filename is included in .gitignore to prevent accidental commits
+max_items=1000
+last_evaluated_key=""
+counter=0
+
+read -p "Enter the AWS region: " region
+read -p "Enter the table name: " table_name
```

Comment:
[dust] Might be cool to have environment variables in our .env that this draws from by default? That way we don't type to type if the usual suspect here is almost always the same 

something like "hit enter to default to your env vars or type in the AWS region"

Reply:
Good thought. These are in the `web/.env` file. I'll see about pulling from there.